pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 # Minimum requirements for the build system to execute.
-requires = ["wheel", "setuptools>=42.0.0", "attrs>=17.1", "setuptools_scm[toml]>=3.4"]
+requires = ["setuptools>=42.0.0", "setuptools_scm[toml]>=3.4"]
```

Comment:
@henryiii In https://github.com/scikit-hep/boost-histogram/pull/394 you bumped the `setuptools_scm` lower bound to `setuptools_scm[toml]>=4.1.2` but have it still as `setuptools_scm[toml]>=3.4` [in `hist`](https://github.com/scikit-hep/hist/blob/56b7fe0b32294e22f5db51bc06e6d4e7059c129d/pyproject.toml#L2). Is there a particular reason for the newer version in `boost-histogram` that we should be updating the lower bounds for as well?